pycontrails/core/met.py

```diff
@@ -2389,9 +2484,12 @@ def originates_from_ecmwf(met: MetDataset | MetDataArray) -> bool:
     - :class:`HRES`
 
     """
-    return met.attrs.get("met_source") in ("ERA5", "HRES") or "ecmwf" in met.attrs.get(
-        "history", ""
-    )
+    if isinstance(met, MetDataset):
+        try:
+            return met.provider_attr == "ECMWF"
```

Comment:
This is really nice - this motivates my comment above, `met.provider` would be a nice access pattern

Reply:
OK nevermind, the `_attr` suffix is probably good for clarity